audio/out/ao_jack.c

```diff
@@ -212,11 +242,10 @@ static int init(struct ao *ao)
 
     ao->samplerate = jack_get_sample_rate(p->client);
 
-    jack_latency_range_t jack_latency_range;
-    jack_port_get_latency_range(p->ports[0], JackPlaybackLatency,
-                                &jack_latency_range);
-    p->jack_latency = (float)(jack_latency_range.max + jack_get_buffer_size(p->client))
-                      / (float)ao->samplerate;
+    p->buffer_size       = ATOMIC_VAR_INIT(0);
+    p->graph_latency_max = ATOMIC_VAR_INIT(0);
```

Comment:
It seems by strict reading of the standard, `atomic_init` or `ATOMIC_VAR_INIT` is needed in theory, except for static or TLS data. That means it you malloc an atomic variable, you need to initialize it with one of those two things, even if the memory was zero-initialized before. This is probably the C committee trying to accommodate really crappy atomics implementation. I say we ignore this, just like most mpv code already ignores this.